:100644 100644 6737792... 29fcdc1... M	data/org.pantheon.noise.desktop.in.in
:100644 100644 e5de4f5... daede01... M	images/CMakeLists.txt
:100644 100644 9249d43... 9249d43... R100	images/icons/128x128/apps/multimedia-audio-player.svg	images/icons/128x128/apps/org.pantheon.noise.svg
:100644 100644 1826fac... 1826fac... R100	images/icons/16x16/apps/multimedia-audio-player.svg	images/icons/16x16/apps/org.pantheon.noise.svg
:100644 100644 afd2d85... afd2d85... R100	images/icons/22x22/apps/multimedia-audio-player.svg	images/icons/22x22/apps/org.pantheon.noise.svg
:100644 100644 090be34... 090be34... R100	images/icons/24x24/apps/multimedia-audio-player.svg	images/icons/24x24/apps/org.pantheon.noise.svg
:100644 100644 7073e3c... 7073e3c... R100	images/icons/32x32/apps/multimedia-audio-player.svg	images/icons/32x32/apps/org.pantheon.noise.svg
:100644 100644 e241736... e241736... R100	images/icons/48x48/apps/multimedia-audio-player.svg	images/icons/48x48/apps/org.pantheon.noise.svg
:100644 100644 3c93624... 3c93624... R100	images/icons/64x64/apps/multimedia-audio-player.svg	images/icons/64x64/apps/org.pantheon.noise.svg

diff --git a/data/org.pantheon.noise.desktop.in.in b/data/org.pantheon.noise.desktop.in.in
index 6737792..29fcdc1 100644
--- a/data/org.pantheon.noise.desktop.in.in
+++ b/data/org.pantheon.noise.desktop.in.in
@@ -4,7 +4,7 @@ _Name=@APP_NAME@
 _GenericName=Music Player
 _Comment=Listen to music
 Exec=noise %U
-Icon=multimedia-audio-player
+Icon=org.pantheon.noise
 Terminal=false
 Categories=Audio;Music;Player;AudioVideo;GNOME;GTK;
 MimeType=x-content/audio-player;x-content/audio-cdda;application/ogg;application/x-extension-m4a;application/x-extension-mp4;application/x-flac;application/x-ogg;audio/3gpp;audio/aac;audio/ac3;audio/AMR;audio/AMR-WB;audio/basic;audio/flac;audio/midi;audio/mp2;audio/mp4;audio/mpeg;audio/ogg;audio/vnd.rn-realaudio;audio/x-aiff;audio/x-ape;audio/x-flac;audio/x-gsm;audio/x-it;audio/x-m4a;audio/x-matroska;audio/x-mod;audio/x-mp3;audio/x-mpeg;audio/x-mpegurl;audio/x-ms-asf;audio/x-ms-asx;audio/x-ms-wax;audio/x-ms-wma;audio/x-musepack;audio/x-opus+ogg;audio/x-pn-aiff;audio/x-pn-au;audio/x-pn-realaudio;audio/x-pn-realaudio-plugin;audio/x-pn-wav;audio/x-pn-windows-acm;audio/x-realaudio;audio/x-real-audio;audio/x-sbc;audio/x-scpls;audio/x-speex;audio/x-tta;audio/x-vorbis;audio/x-vorbis+ogg;audio/x-wav;audio/x-wavpack;audio/x-xm;
diff --git a/images/CMakeLists.txt b/images/CMakeLists.txt
index e5de4f5..daede01 100644
--- a/images/CMakeLists.txt
+++ b/images/CMakeLists.txt
@@ -4,14 +4,14 @@
 
 set (SYSTEM_DEFAULT_THEME ${CMAKE_INSTALL_PREFIX}/share/icons/hicolor)
 
-install (FILES icons/16x16/apps/multimedia-audio-player.svg DESTINATION ${SYSTEM_DEFAULT_THEME}/16x16/apps)
-install (FILES icons/22x22/apps/multimedia-audio-player.svg DESTINATION ${SYSTEM_DEFAULT_THEME}/22x22/apps)
-install (FILES icons/24x24/apps/multimedia-audio-player.svg DESTINATION ${SYSTEM_DEFAULT_THEME}/24x24/apps)
-install (FILES icons/32x32/apps/multimedia-audio-player.svg DESTINATION ${SYSTEM_DEFAULT_THEME}/32x32/apps)
-install (FILES icons/48x48/apps/multimedia-audio-player.svg DESTINATION ${SYSTEM_DEFAULT_THEME}/48x48/apps)
-install (FILES icons/64x64/apps/multimedia-audio-player.svg DESTINATION ${SYSTEM_DEFAULT_THEME}/64x64/apps)
-install (FILES icons/128x128/apps/multimedia-audio-player.svg DESTINATION ${SYSTEM_DEFAULT_THEME}/128x128/apps)
-install (FILES icons/128x128/apps/multimedia-audio-player.svg DESTINATION ${SYSTEM_DEFAULT_THEME}/scalable/apps)
+install (FILES icons/16x16/apps/org.pantheon.noise.svg DESTINATION ${SYSTEM_DEFAULT_THEME}/16x16/apps)
+install (FILES icons/22x22/apps/org.pantheon.noise.svg DESTINATION ${SYSTEM_DEFAULT_THEME}/22x22/apps)
+install (FILES icons/24x24/apps/org.pantheon.noise.svg DESTINATION ${SYSTEM_DEFAULT_THEME}/24x24/apps)
+install (FILES icons/32x32/apps/org.pantheon.noise.svg DESTINATION ${SYSTEM_DEFAULT_THEME}/32x32/apps)
+install (FILES icons/48x48/apps/org.pantheon.noise.svg DESTINATION ${SYSTEM_DEFAULT_THEME}/48x48/apps)
+install (FILES icons/64x64/apps/org.pantheon.noise.svg DESTINATION ${SYSTEM_DEFAULT_THEME}/64x64/apps)
+install (FILES icons/128x128/apps/org.pantheon.noise.svg DESTINATION ${SYSTEM_DEFAULT_THEME}/128x128/apps)
+install (FILES icons/128x128/apps/org.pantheon.noise.svg DESTINATION ${SYSTEM_DEFAULT_THEME}/scalable/apps)
 
 #
 # Fallback icons. These compliment the default "hicolor" theme, but are installed to a
diff --git a/images/icons/128x128/apps/multimedia-audio-player.svg b/images/icons/128x128/apps/org.pantheon.noise.svg
similarity index 100%
rename from images/icons/128x128/apps/multimedia-audio-player.svg
rename to images/icons/128x128/apps/org.pantheon.noise.svg
diff --git a/images/icons/16x16/apps/multimedia-audio-player.svg b/images/icons/16x16/apps/org.pantheon.noise.svg
similarity index 100%
rename from images/icons/16x16/apps/multimedia-audio-player.svg
rename to images/icons/16x16/apps/org.pantheon.noise.svg
diff --git a/images/icons/22x22/apps/multimedia-audio-player.svg b/images/icons/22x22/apps/org.pantheon.noise.svg
similarity index 100%
rename from images/icons/22x22/apps/multimedia-audio-player.svg
rename to images/icons/22x22/apps/org.pantheon.noise.svg
diff --git a/images/icons/24x24/apps/multimedia-audio-player.svg b/images/icons/24x24/apps/org.pantheon.noise.svg
similarity index 100%
rename from images/icons/24x24/apps/multimedia-audio-player.svg
rename to images/icons/24x24/apps/org.pantheon.noise.svg
diff --git a/images/icons/32x32/apps/multimedia-audio-player.svg b/images/icons/32x32/apps/org.pantheon.noise.svg
similarity index 100%
rename from images/icons/32x32/apps/multimedia-audio-player.svg
rename to images/icons/32x32/apps/org.pantheon.noise.svg
diff --git a/images/icons/48x48/apps/multimedia-audio-player.svg b/images/icons/48x48/apps/org.pantheon.noise.svg
similarity index 100%
rename from images/icons/48x48/apps/multimedia-audio-player.svg
rename to images/icons/48x48/apps/org.pantheon.noise.svg
diff --git a/images/icons/64x64/apps/multimedia-audio-player.svg b/images/icons/64x64/apps/org.pantheon.noise.svg
similarity index 100%
rename from images/icons/64x64/apps/multimedia-audio-player.svg
rename to images/icons/64x64/apps/org.pantheon.noise.svg
